src/elements/content-sidebar/AdditionalTabs.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @flow
+ * @file Preview sidebar additional tabs components
+ * @author Box
+ */
+
+import * as React from 'react';
+import AdditionalTab from './AdditionalTab';
+import './AdditionalTabs.scss';
+
+type Props = {
+    tabs: AdditionalTabs,
+};
+
+const AdditionalSidebarTabs = ({ tabs }: Props) => {
+    const sidebarTabs = tabs.map(tabData => <AdditionalTab key={tabData.id} {...tabData} />);
+    return (
+        <React.Fragment>
+            <div className="bcs-nav-separator" />
```

Comment:
Can we wrap this component in a regular `div` and give it a pseudo-element (`::before`) to achieve the visual separation?